README.md

```diff
@@ -15,36 +15,36 @@ Questions or feedback? Join the [Oracle Linux Discussion](https://github.com/ora
 ### Convert CentOS to Oracle Linux
 Looking to switch from CentOS to Oracle Linux? Head over to the [centos2ol repo](/oracle/centos2ol) for a script designed to automatically switch a CentOS instance to Oracle Linux.
 
-### Run Container Images
+### Run container images
```

Comment:
You don't run container images. You build images and you run containers based on images. I